src/WebJobs.Script.WebHost/Management/LinuxSpecialization/BashCommandHandler.cs

```diff
@@ -41,7 +43,7 @@ public BashCommandHandler(IMetricsLogger metricsLogger, ILogger<BashCommandHandl
                             CreateNoWindow = true
                         }
                     };
-                    _logger.LogInformation($"Running: {process.StartInfo.FileName} {process.StartInfo.Arguments}");
+                    _logger.LogInformation(ReplaceStorageSASTokenSecret($"Running: {process.StartInfo.FileName} {process.StartInfo.Arguments}"));
```

Comment:
We already have an existing component [Sanitizer](https://github.com/Azure/azure-functions-host/blob/259db7282f347522e8550f1262cc8fb07e583b0f/src/WebJobs.Script/Sanitizer.cs#L12) that we use in other places for this purpose. Recommend you use that and verify that it removes secrets in your scenario as well. It should.

Reply:
Hi @mathewc  
Thanks! Sanitizer is what I wanted!!!  I'll use it!
The argument can't be share in here. I'll add you on the e-mail thread.

Reply:
I tested locally, and found this case, Sanitizer doesn't detect it. 
I added pattern on Sanitizer.

Reply:
@balag0 - do you see any issues if arguments are not logged?

Reply:
We talked on an e-mail thread. Mathew recommended us to do the following approach. 
* Remove the centralized logging.
* Do the logging on the caller side.

I updated by this commit.

https://github.com/Azure/azure-functions-host/pull/8935/commits/8ffe3fc430414317c9d01008f720bbe26ed95e28